tools/xdp/s2n-quic-xdp/src/mmap.rs

```diff
@@ -19,6 +19,13 @@ pub struct Mmap {
     len: usize,
 }
 
+#[derive(Debug)]
+pub enum MmapOptions {
```

Comment:
Sorry one nitpick on the name here: can we use Options instead? This enum is already in the `mmap` module so the prefix is a bit redundant.